sdk/keyvault/tests.yml

```diff
@@ -10,11 +10,57 @@ resources:
 jobs:
 - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
   parameters:
-    MaxParallel: 1
     ServiceDirectory: keyvault
     EnvVars:
       AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
       AZURE_CLIENT_ID: $(net-keyvault-azure-client-id)
       AZURE_CLIENT_SECRET: $(net-keyvault-azure-client-secret)
       AZURE_KEYVAULT_URL: $(net-keyvault-azure-keyvault-url)
```

Comment:
Don't you still need to switch this to `$[..]` to make it be runtime evaluated? https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops

Reply:
I suppose it wouldn't hurt, but should be unnecessary since the code below actually overwrites that variable and the environment variable. And it seemed to work.